### PR TITLE
#175756592 Created a welcome Page for Visit Africa App

### PR DESCRIPTION
**Pull request by Floride125-art**
#### What does this PR do?
It indicates a welcome page task that I have worked on and some modification including arranging files in specific folders according to the folder structure so as to make it look great.
#### Description of Task to be completed?
The welcome Page task that I worked on is the first page a user see when open the visit Africa app. This page simply summarize what the visit Africa app do and allow user to continue to the next page where she or he can be able to register and create an account with  us. 
#### How should this be manually tested?
clone it  
run flutter pub get to install dependencies  
run flutter run to start
#### Any background context you want to provide?
As the user you should click on continue button to be able to proceed on the next pages of Visit Africa App.
#### What are the relevant pivotal tracker stories?
#175756592
#### Screenshots (if appropriate)
welcomeIMg

#### Questions:
N/A

### CODE REVIEW

File: lib/welcomePage.dart [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      appBar: AppBar(
+        title: Text('Visit Africa'),
+        centerTitle: true,
+        backgroundColor: Color(0xffB15C1E),
+      ),
+      body: Container(
+        child: Stack(
+          children: <Widget>[
+            Image.asset(
+              "assets/cover.png",
```
Comment (VedaDom) [inline lib/welcomePage.dart]: Can you put images in the images folder? `assets/images/example.png`

**Comment by dianahub250** [comment]
Great work! You can proceed!

**Review by Sandrine21 — APPROVED** [review]
The work is well done. I am good with it.

File: lib/signup.dart [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import 'package:flutter/cupertino.dart';
+
+class Signup extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Visit Africa'),
+        centerTitle: true,
+        backgroundColor: Color(0xffB15C1E),
```
Comment (VedaDom) [inline lib/signup.dart]: Why do we have this `app bar ` here not in the original design? It kills everything. what do you think about removing it?

File: lib/welcomePage.dart [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+              width: 1100,
+              height: 560,
+              fit: BoxFit.cover,
+            ),
+            Padding(
+              padding: const EdgeInsets.only(top: 470.0, left: 30.0),
+              child: Text(
+                'Welcome',
```
Comment (VedaDom) [inline lib/welcomePage.dart]: Aren't we supposed to have Welcome to **visit Africa** ? what changed your mind?

File: lib/welcomePage.dart [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+              child: Text(
+                'Join us to explore the whole africa by visiting best places',
+                style: TextStyle(
+                  fontSize: 13.0,
+                  color: Color(0xff7A7A7A),
+                ),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.only(top: 630.0, left: 30.0),
```
Comment (VedaDom) [inline lib/welcomePage.dart]: Can you find another way to move this to bottom?

**Review by VedaDom — CHANGES_REQUESTED** [review]

**Review by Rita-12397 — APPROVED** [review]

**Review by dianahub250 — APPROVED** [review]
This is the Good work!

**Review by mujohn26 — COMMENTED** [review]
LGTM

**Review by VedaDom — APPROVED** [review]

**Pull request merged**